hash/msx1_cart.xml

```diff
@@ -18384,6 +18480,300 @@ legacy FM implementations cannot find it.
+		<info name="developer" value="Impulse 9"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="italian stallion - impulse 9 (2007).rom" size="0x8000" crc="904e0edd" sha1="64f921c9c57aedb47663a269046d4c51388f41fb" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jetpac">
+		<description>J.E.T.P.A.C.</description>
+		<year>2009</year>
+		<publisher>MSX Cartridge Shop</publisher>
+		<info name="developer" value="Imanok"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="j.e.t.p.a.c. - imanok (2009).rom" size="0x8000" crc="38ad7112" sha1="cc13bac2f50f58bc466d605839187cf09dbe269c" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jaasta">
+		<description>Jäästä</description>
+		<year>2020</year>
+		<publisher>MSXDev</publisher>
+		<info name="developer" value="Uninteresting"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="jaasta - uninteresting (2020).rom" size="0x8000" crc="77380a9d" sha1="e5e0da269094bd2bf1e2aadd92d6523518f4de89" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jawbreak">
+		<description>Jawbreaker II</description>
+		<year>2014</year>
+		<publisher>MSXDev</publisher>
+		<info name="developer" value="Maggoo"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x4000">
+				<rom name="jawbreaker ii - maggoo (2014).rom" size="0x4000" crc="4137905f" sha1="ae3eb3af124b36878ab4f189abfc3737bfd3d56a" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jumping">
+		<description>JumpinG - The Tutorial Game (English)</description>
+		<year>2011</year>
+		<publisher>MSXDev</publisher>
+		<info name="developer" value="Dimension Z"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="jumping - dimension z (2011) (english).rom" size="0x8000" crc="7f492f09" sha1="dbea257c0f3cc4d46a94d2db30370df64eb02221" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jumpings" cloneof="jumping">
+		<description>JumpinG - The Tutorial Game (Spanish)</description>
+		<year>2011</year>
+		<publisher>MSXDev</publisher>
+		<info name="developer" value="Dimension Z"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="jumping - dimension z (2011) (spanish).rom" size="0x8000" crc="d19a6356" sha1="74f7fba8a016e4ead892c2db5970028ed1aa4fa2" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="jumpingsd" cloneof="jumping">
+		<description>JumpinG - The Tutorial Game (Spanish, demo)</description>
+		<year>2011</year>
+		<publisher>&lt;homebrew&gt;</publisher>
```

Comment:
I have not found where the software was released or who released it so I have kept this as '<homebrew>'